src/model/cytoscape-syncher.js

```diff
@@ -67,22 +69,22 @@ export class CytoscapeSyncher {
       }
     };
 
-    if (isServer() && USE_COUCH_AUTH) {
+    if (this.isServer() && USE_COUCH_AUTH) {
       const auth = remotePouchOptions.auth = {};
 
       auth.username = COUCHDB_USER;
       auth.password = COUCHDB_PASSWORD;
     }
 
-    if (this.editable() || isServer()) {
-      this.localDb = new PouchDB(this.dbName, { adapter: 'memory' }); // store in memory to avoid multitab db event noise
+    if (this.editable() || this.isServer()) {
+      this.localDb = new PouchDB(this.dbName + Math.random(), { adapter: 'memory' }); // store in memory to avoid multitab db event noise
```

Comment:
N.b. the random number is needed to prevent two testing instances from pointing to the same in-memory database.  Randomising the name ensures that each instance is backed uniquely.  It may be better to use something like a UUID here to ensure that collisions are avoided.